src/connector/src/parser/mod.rs

```diff
@@ -644,14 +644,14 @@ pub trait ByteStreamSourceParser: Send + Debug + Sized + 'static {
 }
 
 #[try_stream(ok = Vec<SourceMessage>, error = ConnectorError)]
-async fn ensure_largest_at_rate_limit(stream: BoxSourceStream, rate_limit: u32) {
+async fn ensure_max_chunk_size(stream: BoxSourceStream, max_chunk_size: usize) {
```

Comment:
This function is actually not needed anymore after my future PR, will remove then.

Reply:
Is the future PR still WIP? I'd like to review that PR to see the changes.

Reply:
Still in progress